views/theme/assets/scss/app.scss

```diff
@@ -5,6 +5,7 @@
 @import "components/button";
 @import "components/card";
 @import "components/chip";
+@import "components/datetime";
```

Comment:
What do you think about calling this file `flatpickr` instead? It might make it easier to find in the future.

Reply:
no difference to me – flatpickr makes sense, datetime matches COPRL. the file's currently specific to flatpickr, though it could include datetime things in the future. _shrug_

Reply:
I think `datetime.scss` would make sense for our own styles, but we're specifically overriding flatpickr styles here.